### DISCUSSION TITLE
2022.09.10 브리핑

### DISCUSSION THREAD

**Discussion by itmakesmesoft** [itmakesmesoft/Steady-Study#74] [Self-Assessment]
이번 주 목표를 이루기 위해 어떤 노력을 했는지 스스로 평가해보고, 점수를 매겨주세요.
ex)
이번 주 목표 : 기술 블로그에 배운 내용 정리하기

점수 : 80점
이유 : (생략)

**Comment by itmakesmesoft** [comment]
이번 주 목표 : 자바로 민코딩 3문제 도전. 간헐적 단식 도전, 온라인 수업일 산책 1시간 이상 하기. velog로 글 옮기기

점수 : 70점
이유 : 
- 자바로 민코딩 문제 3문제 도전 - succeed
- 간헐적 단식 - fail
- 온라인 수업일 산책 1시간 이상 하기 - succeed
- velog로 글 옮기기 - succeed

  2줄 요약
  운동을 하고도 살이 안빠졌다. 
  운동으로 소모한 칼로리보다 더 많은 과자를 먹었다.

[velog](https://velog.io/@dmsgur7112)    |    [인증사진](https://github.com/itmakesmesoft/Steady-Study/blob/main/%EC%9D%B4%EC%9D%80%ED%98%81/2022.09.10%20%EC%9D%B8%EC%A6%9D%EC%82%AC%EC%A7%84.md)

**Comment by YoungSeok222222** [comment]
이번 주 목표: 1일 1알고리즘, Dajngo 교육과정 잘 따라가기
점수 : 50점
이유:

- 1일 1알고리즘 : 절반 성공=> 하루에 한 개씩 하기가 목표였지만, 못한 날과 1개 이상 푼 날들이 섞였기 때문
- Django 교육과정 잘 따라가기 => 과제 완벽히 하기였으면 0점이겠지만, 목표를 낮게 잡아 교육과정 따라가기였기 때문에 해당 목표는 성공하였습니다.

**Comment by HyeseungNA** [comment]
이번주 목표: 1일 1 알고리즘, 민코딩 재귀풀기, 수업 복습하기
점수 : 50점
이유:
1일 1알고리즘은 성공했습니다.
민코딩 재귀풀기 절반 풀었습니다.
수업은 매번 복습했으나 피피티 없이 혼자 코드를 짤 수 없기 때문에 성공이라고 할 수 없을 것 같습니다.

**Comment by giokim12** [comment]
이번주 목표 : 민코딩 매일 3문제 이상(토요일까지 18문제), 장고 수업 따라가기+gitignore 마스터하기, 야식 주 1회
점수: 50점
민코딩 18문제는 풀었지만 매일 3문제 푼게 아니라 벼락치기로 18문제 풀었습니다 ㅠ 매일 시간 내는게 쉽지 않네요
장고는... (말잇못) ... 빛기륜형 죄송해요
야식은 어쩌다보니 약속이 파토나서 안먹었어용

**Comment by swjiae** [comment]
이번 주 목표: swea 과제 완료하기, 주 3회 1시간 이상 운동하기
점수 : 0점
이유: 알고리즘 1도 못풀었고 운동도 안했다ㅠㅠ..

지난 2주간 좀 해이해졌지만 오늘부터 정신차리겠습니다!

**Comment by sooyeonlee127** [comment]
이번 주 목표: 블로그 글 7개 이상 올리기, 정처기 필기 강의 1강 이상 듣기
점수 : 80점
이유: 블로그 글 올리기 11개, 정처기 필기는 OT 영상만 봤습니다..
블로그 글 올리기는 초과 달성했으니까 80점 드리겠습니다..

알고리즘 주간이 돌아오니, 정신 다시 잡고 열심히 해보겠습니다.. 아자 아자..!!!!

**Comment by moonthree** [comment]
이번 주 목표 : 7일 7 백준, HTML/CSS/JS로 게임 만들기 클론코딩, 복싱 2회 이상 출석
점수 : 50점
이유 : 백준 2개, 클론코딩 x, 복싱 2회지만 추석에 건강해졌으니 50점 드리겠습니다...
